### PR TITLE
MEN-2966: Align variable default descriptions with the Debian list.

### PR DESCRIPTION
**Pull request by kacf**
Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>

### CODE REVIEW

File: 04.Artifacts/01.Yocto-project/99.Variables/docs.md [hunk 1]
```diff
@@ -337,28 +397,36 @@ See also [`MENDER_MTDIDS`](#mender-mtdids).
...
 #### MENDER_NAND_FLASH_PAGE_SIZE
 
+> Value: 2048 (default)
+
 This variable sets the page size, in bytes, of the NAND flash on the device, and
 is used to calculate parameters for the UBI volumes.
 
 
-#### MENDER_PARTITIONING_OVERHEAD_MB
+#### MENDER_PARTITIONING_OVERHEAD_KB
```
Comment (oleorhagen) [inline 04.Artifacts/01.Yocto-project/99.Variables/docs.md]: Damn, well done catching this halfway through this repetetive task!

File: 04.Artifacts/01.Yocto-project/99.Variables/docs.md [hunk 2]
```diff
@@ -441,12 +522,16 @@ for more information.
 
 #### MENDER_STORAGE_PEB_SIZE
 
+> Value: 8388608 (default)
+
 Holds the size, in bytes, of the physical erase blocks (PEBs) on the Flash
 device.
 
 
 #### MENDER_STORAGE_TOTAL_SIZE_MB
```
Comment (oleorhagen) [inline 04.Artifacts/01.Yocto-project/99.Variables/docs.md]: We always write `MB` when we are really meaning `MiB` right?

File: 04.Artifacts/01.Yocto-project/99.Variables/docs.md (hunk 1 above)
Comment (kacf) [reply 04.Artifacts/01.Yocto-project/99.Variables/docs.md]: Simple, I was grepping to find the default, and it didn't show up!

File: 04.Artifacts/01.Yocto-project/99.Variables/docs.md (hunk 2 above)
Comment (kacf) [reply 04.Artifacts/01.Yocto-project/99.Variables/docs.md]: Yes, this is adopted from Yocto.

**Pull request merged**